### ISSUE TITLE
bug: unable to find app selections and patches

### ISSUE THREAD

**Issue by sssh7710** [revanced/revanced-manager#832]
### Type

Error while running the manager

### Bug description

While browsing after updating to a new release version, I noticed an issue where I couldn't find app selections and patches

### Steps to reproduce

1. Install the Manager app 0.1.0 version (I don't know why, but it says 1.0.0 after installation)
2. Enter the app and enter the list of apps to patch
3. First, select a genuine app file
4. I can't find a list of patches for this app

### Android version

Android 12,13

### Manager version

0.1.0 (displayed as 1.0.0 after installation)

### Target package name

Youtube

### Target package version.

18.16.37

### Installation type

Non-root

### Patches selected.

Patch Selection Screen Annotated

### Device logs (exported using Manager settings).

```shell
I don't know how to open the log, so I add it to the bottom
```

### Screenshots or videos

![Screenshot_20230430-135404](https://user-images.githubusercontent.com/98168192/235336680-3043db33-afde-40ec-a3a3-11453ce01a50.jpg)
![Screenshot_20230430-135417](https://user-images.githubusercontent.com/98168192/235336682-d2941b26-0bb7-437a-b12e-9c4712f6cc28.jpg)

### Additional context

[revanced-manager_logcat_20230430140540084708.log](https://github.com/revanced/revanced-manager/files/11360620/revanced-manager_logcat_20230430140540084708.log)

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [x] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.
- [X] The issue is related solely to the ReVanced Manager and not related to patching errors or patches

**Comment by prat89** [comment]
+1

**Comment by mik1011984** [comment]
Same issue here. Also the app thinks there's an update available for manager even though that dev build is likely exactly what was released as v0.1.0 (or 1.0.0 as it says in the settings).

**Comment by sssh7710** [comment]
Is it the same for the test version?

**Comment by sssh7710** [comment]
The test version is the same

**Comment by SodaWithoutSparkles** [comment]
Duplicate of #805

?

**Comment by oSumAtrIX** [comment]
This is different from #805, because this issue mentions an issue with the app selection screen.

**Comment by ILoveOpenSourceApplications** [comment]
Question: All the users here who said they're having the same issue, are you all using ReVanced in the sources or are you all RVX users?

**Comment by SodaWithoutSparkles** [comment]
Suggestion: Try nuking everything and restart. If you have installed any patched apps before, export the keystore. 

1. Update manager to ≥v1.0.0
2. [export keystore](https://sodawithoutsparkles.github.io/revanced-troubleshooting-guide/troubleshoot/02-manager/04/)
3. Delete revanced manager, install again
4. [import keystore](https://sodawithoutsparkles.github.io/revanced-troubleshooting-guide/troubleshoot/02-manager/05/)

**Comment by SodaWithoutSparkles** [comment]
@ILoveOpenSourceApplications discord user K4T#0170 reported this bug in the support channel. After sending him this guide, he found out that he was using RVX. Reverting back to normal fixed it.

**Comment by ILoveOpenSourceApplications** [comment]
Yeah I figured. RVX patches needs to updated to be compatible with the latest manager. Until so either RVX users will have to revert back to 0.0.57 or wait till inotia00 updates the compatibility. Either way this issue isn't related to the manager anymore and hence will be closed.

**Comment by kitadai31** [comment]
No.
This is a problem that occurs not only with Extended, but with custom sources in general.
Another custom source (like my kitadai31/revanced-patces-android6-7) is also not available in v1.0.0.

ReVanced Patcher dependencies and API spec should not have changed in v1.0.0.

Please refer to this conversation.
https://discord.com/channels/952946952348270622/952946952348270626/1104170349593120769

It seems that one of the developer knows this issue. (Time is UTC+9)
> CossaleX — 2023/05/06 07:20
> i would guess issue is because of new caching impl

**Comment by ILoveOpenSourceApplications** [comment]
Well if that's the case then I'll leave this open for one of the developers to look into.

**Comment by Aunali321** [comment]
fixed in https://github.com/revanced/revanced-manager/pull/857/commits/3fe5882145cdc6dfb0e3b3cdd5008c834f935a3a

**Comment by N367B** [comment]
Hello, I'm having the sames issues while patching any app with revanced manager (I downloaded the latest version), I'm getting "No applications found" when trying to select an app I'm also getting "No patches found for the selected app" when trying with an apk. I tried with two different devices. 

What should I do?

![Screenshot_20231018-185556_ReVanced Manager](https://github.com/ReVanced/revanced-manager/assets/42220691/b2a077d1-e751-4c5d-8d59-7c4f79b5a0eb)

![Screenshot_20231018-185552_ReVanced Manager](https://github.com/ReVanced/revanced-manager/assets/42220691/3658fd57-d0af-4f98-96fc-bdf373afe218)
